ddtrace/contrib/algoliasearch/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""
+The Algoliasearch__ integration will add tracing to your Algolia searches.
+
+::
+
+    from ddtrace import patch_all
+    patch_all()
+
+    from algoliasearch import algoliasearch
+    client = alogliasearch.Client(<ID>, <API_KEY>)
+    index = client.init_index(<INDEX_NAME>)
+    index.search("your query", args={"attributesToRetrieve": "attribute1,attribute1"})
+
+Configuration
+~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.algoliasearch['collect_query_text']
+
+   Whether to pass the text of your query onto Datadog. Since this may contain sensitive data it's off by default
+
+   Default: ``False``
+
+.. __: https://www.algolia.com
+"""
```

Comment:
Does anyone know whether this gets rendered somewhere?

Reply:
It does.

The current released docs live here: http://pypi.datadoghq.com/trace/docs/

To build them locally:

```
rake docs
open docs/_build/html/index.html
```

Reply:
Thanks! Just added another commit to make algoliasearch render correctly.